src/NServiceBus.PersistenceTesting/.editorconfig

```diff
@@ -4,4 +4,7 @@
 dotnet_diagnostic.CA2007.severity = none
 
 # Justification: net461
-dotnet_diagnostic.IDE0074.severity = none
\ No newline at end of file
+dotnet_diagnostic.IDE0074.severity = none
+
+# IDE0028: Simplify collection initialization
```

Comment:
**Nitpick**: Can you please update this comment to have the same format as the others, meaning: "Justification:..."